src/librustc_resolve/late/diagnostics.rs

```diff
@@ -295,6 +295,7 @@ impl<'a> LateResolutionVisitor<'a, '_> {
                     }
                 }
 
+                err.set_span(call_span);
```

Comment:
Sorry, I realised I misread the output before and flipped the hint/error spans in my head. They were right beforehand: I'm sorry for wasting your time! Let's revert the last commit, and then I'll approve.

Reply:
No worries :) Reverted last commit